Source/WebCore/page/LocalDOMWindow.cpp

```diff
@@ -1690,9 +1690,13 @@ Ref<CSSStyleDeclaration> LocalDOMWindow::getComputedStyle(Element& element, cons
 {
     std::optional<PseudoId> pseudoId = PseudoId::None;
     // FIXME: This does not work for pseudo-elements that take arguments (webkit.org/b/264103).
-    if (pseudoElt.startsWith(":"_s))
+    if (pseudoElt.startsWith(":"_s)) {
```

Comment:
Nit: if you invert this you can have less nesting. And either way you can remove the upfront pseudoId declaration.

Reply:
Good suggestion, done